rules-tests/Php81/Rector/FuncCall/ClosureFromCallableToFirstClassCallableRector/Fixture/class_callable_array.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
```

Comment:
Add namespace in the fixture when possible like other rules tests, except on purpose